localization/RingTimeSync.py

```diff
@@ -8,12 +8,12 @@
 from RingBufferFuncs import RingBuffer
 
 # Hand-written data for test
-t_std = np.array([1, 2, 3, 4, 8, 10])
-t_unsync = np.array(
-    [1.95, 2.05, 2.5, 2.8, 2.95, 3.2, 3.9, 4.2, 4.4, 4.8, 5, 6, 10.5])
-
 #t_std = np.array([1, 2, 3, 4, 8, 10])
-#t_unsync = np.array([1.95, 2.05, 2.5, 2.8, 2.95, 3.2, 3.9, 4.2, 4.3, 4.8, 5, 9.95, 10.05])
+#t_unsync = np.array(
+    #[1.95, 2.05, 2.5, 2.8, 2.95, 3.2, 3.9, 4.2, 4.4, 4.8, 5, 6, 10.5])
+
+t_std = np.array([1, 2, 3, 4, 8, 10])
+t_unsync = np.array([1.95, 2.05, 2.5, 2.8, 2.95, 3.2, 3.9, 4.2, 4.3, 4.8, 5, 9.95, 10.05])
```

Comment:
Have not seen these magic number usage... Could you please add some comment for generating these magic number?

Reply:
Done, thx for ur advice.